libs/dh/charges/feature-prices/src/lib/search-result/dh-charges-prices-result.component.html

```diff
@@ -0,0 +1,124 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *transloco="let transloco; read: 'charges.prices'">
+  <div class="card-container">
+    <div class="header-container">
+      <h3>{{ this.searchResult.length }} {{ transloco("resultLength") }}</h3>
+
+      <div class="pagination-container">
+        <!-- Download -->
+        <watt-button
+          variant="text"
+          *dhFeatureFlag="'start_wholesale_process_feature_flag'"
+        >
+          <watt-icon
+            name="openInNew"
+            label="openInNew"
+            [size]="iconSizes.Medium"
+          ></watt-icon>
+          {{ transloco("Download") }}
+        </watt-button>
+
+        <!-- Pagination -->
+        <mat-paginator
+          [pageSizeOptions]="[5, 10, 20, 50]"
+          pageSize="10"
```

Comment:
suggestion: use `[50, 100, 150, 200, 250]` and`100` as default according to UX (will probably be refined later).